bundle/src/main/java/com/adobe/acs/commons/synthesizedsling/SynthesizedUtil.java

```diff
@@ -0,0 +1,75 @@
+package com.adobe.acs.commons.synthesizedsling;
+
+import com.day.cq.commons.feed.StringResponseWrapper;
+import com.day.cq.wcm.api.components.IncludeOptions;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.request.RequestDispatcherOptions;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.Map;
+
+public final class SynthesizedUtil {
+
+    private SynthesizedUtil() {}
+
+    public static Resource buildSynthesizedResource(ResourceResolver resourceResolver, String path,
```

Comment:
Is there value in passing in a "path";  is there any advantage to providing it vs hard-coding it to something like "/var/acs-commons/synthesized" in this impl? 

Isnt it the properties and resourceType that matter?